src/main/java/com/minecolonies/items/ItemBuildTool.java

```diff
@@ -27,14 +27,13 @@ public ItemBuildTool()
     @Override
     public EnumActionResult onItemUse(ItemStack stack, EntityPlayer playerIn, World worldIn, BlockPos pos, EnumHand hand, EnumFacing facing, float hitX, float hitY, float hitZ)
     {
-        if (!worldIn.isRemote)
+        playerIn.addStat(ModAchievements.achievementWandOfbuilding);
+        if (worldIn.isRemote)
         {
-            return EnumActionResult.FAIL;
+            MineColonies.proxy.openBuildToolWindow(pos.offset(facing));
         }
 
-        playerIn.addStat(ModAchievements.achievementWandOfbuilding);
-        MineColonies.proxy.openBuildToolWindow(pos.offset(facing));
-        return EnumActionResult.FAIL;
+        return EnumActionResult.SUCCESS;
```

Comment:
Success means we have worked on that interact and it is done, so forge knows that something worked. A fail is used for not allowed stuff. and pass is used for spectating events.